scribe/src/main/scala/io/chrisdavenport/log4cats/scribe/ScribeLogger.scala

```diff
@@ -0,0 +1,55 @@
+package io.chrisdavenport.log4cats.scribe
+
+import io.chrisdavenport.log4cats.Logger
+import _root_.scribe.{Logger => Base,  Level, LogRecord}
+import cats.effect.Sync
+
+object ScribeLogger {
+
+  def fromLogger[F[_]: Sync](logger: Base): Logger[F] = new Logger[F]{
+
+    // This May Be a Very Bad Approach...
+    def isTraceEnabled: F[Boolean] = 
+      Sync[F].delay(
+        logger.modifiers.find(mod => mod(LogRecord.simple("", "", "", None, None, Level.Trace)).isDefined).isDefined
```

Comment:
Still likely a terrible idea, but at least has the chance to be succesful.

Reply:
I think we need to look at the `logger.handlers`. I feel like there should be a better way to write this than:
```scala
logger.handlers.exists {
  _.minimumLevel
    .map(_ <= Level.Trace)
    .getOrElse(true)
}.getOrElse(false)
```
or:
```scala
(for {
  handler <- logger.handlers
  minLevel <- handler.minimumLevel.toList
  if minLevel <= Level.Trace
} yield ()).nonEmpty
```
but my brain's too blasted at the moment :laughing:

Reply:
`minimumLevel` does not appear to be a value of the handler, only in the creation of the handler.